packages/web3-core/test/unit/web3_request_manager.test.ts

```diff
@@ -228,7 +228,7 @@ describe('Web3RequestManager', () => {
 			});
 		});
 
-		it('should throw error if can not detect the provider', () => {
+		it.skip('should throw error if can not detect the provider', () => {
```

Comment:
As mentioned above if no provider is provided it should throw error.

Reply:
IPC provider is just string. how we can understand is it the correct IPC string or not?

Reply:
Named pipe can be any string but for now we can condition here to accept specific format and can use regex for its check. and add doc comment on function for valid IPC provider strings.